cmd/scollector/main.go

```diff
@@ -358,3 +327,176 @@ func printPut(c chan *opentsdb.DataPoint) {
 		slog.Info(string(b))
 	}
 }
+
+func toToml(fname string) {
```

Comment:
I'd say leave it in for a few weeks or until we add a new config directive, then take it out. Not intended for new configs or long term work.